common/src/main/java/hunternif/mc/impl/atlas/structure/Village.java

```diff
@@ -1,13 +1,13 @@
 package hunternif.mc.impl.atlas.structure;
 
 import hunternif.mc.impl.atlas.AntiqueAtlasMod;
-import net.minecraft.text.TranslatableText;
-import net.minecraft.world.gen.feature.StructureFeature;
+import net.minecraft.text.Text;
+import net.minecraft.world.gen.structure.StructureType;
 
 public class Village {
     public static void registerMarkers() {
         if (AntiqueAtlasMod.CONFIG.autoVillageMarkers) {
-            StructureHandler.registerMarker(StructureFeature.VILLAGE, AntiqueAtlasMod.id("village"), new TranslatableText("gui.antiqueatlas.marker.village"));
+            StructureHandler.registerMarker(StructureType.JIGSAW, AntiqueAtlasMod.id("village"), Text.translatable("gui.antiqueatlas.marker.village"));
```

Comment:
This isn´t a good idea. Many mods will use the JIGSAW type; hence this may end up adding a gazillion village markers at places where some random mod generated stuff. I´d rather remove village markers entirely.

Reply:
Great point! added a structure tag based map instead in a61ff06, those are lazy-loaded so it works a charm

![image](https://github.com/AntiqueAtlasTeam/AntiqueAtlas/assets/55819817/666f7e49-1474-4ad1-896c-3249e60a7163)